app/src/main/java/com/lagradost/cloudstream3/utils/ExtractorApi.kt

```diff
@@ -871,8 +873,10 @@ val extractorApis: MutableList<ExtractorApi> = arrayListOf(
     Streamlare(),
     VidSrcExtractor(),
     VidSrcExtractor2(),
+    VidSrcTo(),
     PlayLtXyz(),
     AStreamHub(),
+    AnyVidplay(""),
```

Comment:
This makes no sense, a extractor with no url?

Reply:
I get that, but by adding AnyVidplay to the extractor list with no url it messes up some shit